examples/recommender_system/recommender_system_env.py

```diff
@@ -0,0 +1,35 @@
+from typing import Dict, Tuple
+import gym
+import gym.spaces
+import numpy as np
+
+
+gym.envs.registration.register(
+    id='RecommenderSystemEnv-v1',
+    entry_point='examples.recommender_system.recommender_system_env:RecommenderSystemEnv'
+)
+
+
+class RecommenderSystemEnv(gym.Env):
+    
+    def __init__(self, preferences: Dict) -> None:
+        super().__init__()
+        self.content_tags = list(preferences.keys())
+        self.preferences = preferences  # preferences as probability of enjoying the content
+        self.action_space = gym.spaces.Discrete(len(self.content_tags))
+        self.observation_space = gym.spaces.Space()
+    
+    def reset(self, seed: int = None) -> Tuple[gym.spaces.Dict, Dict]:
+
+        self.seed = seed if seed else np.random.randint(1000)
+        super().reset(seed=self.seed)
+        np.random.seed(self.seed)
+
+        return 0, {}
```

Comment:
W zasadzie można zwracać `None` zamiast `0` - jest to czytelniejsze, że nie ma wtedy żadnego stanu. Chyba że gym to jakoś blokuje i trzeba coś zwrócić?

Reply:
można, jak najbardziej